docs/user-guide/install.md

```diff
@@ -158,3 +158,53 @@ Windows service vdvs uninstalled successfully!
 ```
 PS C:\> Set-ExecutionPolicy Unrestricted -Scope Process -Force
 ```
+
+## Upgrade Instructions
+
+The volume driver can be upgraded or reinstalled any time with no impact to the user data. User data in volumes existing before an upgrade of the volume driver is retained as it is and is accessible post an upgrade.
```

Comment:
Please reword this:`User data in volumes existing before an upgrade of the volume driver is retained as it is and is accessible post an upgrade.`

something like: User data in volumes stays intact and remains accessible post upgrade.